addons/website/static/src/scss/website.backend.scss

```diff
@@ -429,6 +429,22 @@
 .o_theme_install_loader_container {
     background-color: rgba($o-shadow-color, .9);
     pointer-events: all;
+    font-size: 4.5rem;
+    justify-content: space-evenly;
+
+    &, & > * {
```

Comment:
why & > *

Reply:
Added because `.o_theme_install_loader_tip` doesn't get applied style from parent (updated now to have cleaner CSS) 
Thanks!